docs/ref/django-admin.txt

```diff
@@ -405,6 +405,13 @@ it because ``True`` is its default value).
 
 Specifies the database to introspect. Defaults to ``default``.
 
+.. django-admin-option:: --include-views
+
+If this option is provided, both database tables and views are inspected.
+Without `--include-views`, only database tables are inspected, although views
+can be inspected individually if they are named as positional arguments to
+``inspectdb``.
+
```

Comment:
Just thinking we need a 

```
.. versionadded:: 2.1
```

as [per the checklist](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/submitting-patches/#new-features)

Reply:
good point! I've just checked that in. Thanks for the suggestion.